mantle/cosa/schema_test.go

```diff
@@ -0,0 +1,81 @@
+package cosa
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+var (
+	testMeta = []string{
+		"../../fixtures/fcos.json",
```

Comment:
I don't have a better idea but not thrilled about having relative paths here.

Reply:
The big :hammer: is to move all of the `mantle/` code to the toplevel, i.e. coreos-assembler becomes a direct mix of Go/Python/shell/etc.  And we rename `mantle/` to `cosa/` or something?